app/models/attendance_record.rb

```diff
@@ -15,15 +15,36 @@ def self.set_attended(meeting, user, value)
     end
     return false
   end
-  
+
   # returns the attendance record for a user attending the meeting
   # if no record is found, returns nil
-  def self.find_record(meeting, user) 
+  def self.find_record(meeting, user)
     ce = CommitteeEnrollment.where(committee_id: meeting.committee_id).and(CommitteeEnrollment.where(user_id: user.id)).take
     unless ce.nil?
       return self.where(meeting_id: meeting.id).and(self.where(committee_enrollment_id: ce.id)).take
     end
     return nil
   end
 
-end
\ No newline at end of file
+  #returns the total number of absences for a user
+  def self.find_total_absences(user)
+    absence_count = 0
+    Meeting.all.each do |meeting|

```

Comment:
Instead of looping through each meeting then finding the record for that meeting that belongs to a user, would it be possible to just search the AttendanceRecord for records belonging to that user? 

Rails has a .count function, so if we can somehow just do AttendanceRecord.where(user = user).count, it would be a lot more efficient.
For now this works though, we can come back to it later.